Agents/AQMeshInputAgent/feature-info-queries/aqmesh_meta.sparql

```diff
@@ -0,0 +1,29 @@
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX geo:	<http://www.opengis.net/ont/geosparql#>
+PREFIX xsd:	<http://www.w3.org/2001/XMLSchema#>
+PREFIX carpark: <https://www.theworldavatar.com/kg/ontocarpark/>
+PREFIX building: <https://www.theworldavatar.com/kg/Building/>
+PREFIX bot: <https://w3id.org/bot#>
+PREFIX ontobim: <https://www.theworldavatar.com/kg/ontobim/>
+SELECT ?Property ?Value ?Unit WHERE {
+  SERVICE <http://sgteststack-blazegraph:8080/blazegraph/namespace/aqmesh/sparql> {
```

Comment:
usually we use service to combine results from ontop, is this needed here?

Reply:
Removed